packages/helpers/scripts/ast_grep.js

```diff
@@ -65,8 +63,10 @@ export function ast_grep() {
                 report_export_mismatch(tree.filename(), match);
             }
 
+            const export_as_lodash_length = `export { ${func_name} as _ }`.length;
+
             const export_start = match.range().start.index;
-            const export_end = export_start + export_lentgh;
+            const export_end = export_start + export_as_lodash_length;
```

Comment:
You should directly use the end index of the match.

Reply:
This is doing same as:

```js
source.prependLeft(range.start.index, `exports._ = exports.${func_name} = `);
```

If I use `match.range().end.index` it throws as:

```
file:///Users/devjiwonchoi/DevLife/forks/swc/node_modules/magic-string/dist/magic-string.es.mjs:761
                if (end > this.original.length) throw new Error('end is out of bounds');
                                                      ^

Error: end is out of bounds
    at MagicString.update (file:///Users/devjiwonchoi/DevLife/forks/swc/node_modules/magic-string/dist/magic-string.es.mjs:761:41)
    at file:///Users/devjiwonchoi/DevLife/forks/swc/packages/helpers/scripts/ast_grep.js:70:20
```

Reply:
Ah, you're right, sorry.